Documentation/creating-new-package.md

```diff
@@ -0,0 +1,29 @@
+# Creating new .NET IoT package
+
+This is a list of steps which need to happen in order to create new package version of .NET IoT.
+
+## Steps
+
+- regenerate device listing (`dotnet run` on https://github.com/dotnet/iot/tree/master/tools/device-listing and fix all warnings)
+  - this always requires manual review
+- ensure all PRs meant for this release are merged
+- ideally until done do not merge any new PRs (technically not required but will avoid confusion and accidental errors)
+- [Internal] create new package:
```

Comment:
Not sure I understand your Internal tag only on these sub-section. Pretty much all instructions need to be performed by internal folks, so I don't understand why those aren't branded as internal too, for example, the push package to NuGet step.